tools/peptideshaker/peptide_shaker.xml

```diff
@@ -316,99 +483,189 @@
+            <discover_datasets pattern="__name_and_ext__" directory="output_mgf" ext="mgf"/>
+        </collection>
+        <collection name="output_cui" type="list" label="${tool.name} on ${on_string}: Cui" >
+            <filter>exporting_options['zip_conditional']['zip_output_boolean'] is True and exporting_options['zip_conditional']['export_mgf_when_zip'] is True</filter>
+            <discover_datasets pattern="__name_and_ext__" directory="output_cui" ext="cui"/>
+        </collection>
+
+        <!-- Follow-up analysis -->
+        <collection name="output_spectra" type="list" label="${tool.name} on ${on_string}: Spectra" >
+            <filter>(exporting_options['zip_conditional']['zip_output_boolean'] is False or (exporting_options['zip_conditional']['zip_output_boolean'] is True and exporting_options['zip_conditional']['export_reports_when_zip'] is True) ) and exporting_options['followup_conditional']['followup_conditional_selector'] == 'yes' and exporting_options['followup_conditional']['followup_export_spectra'] != 'off' </filter>
+            <discover_datasets pattern="__name_and_ext__" directory="output_reports" ext="mgf"/>
+        </collection>
+        <data format="txt" name="output_proteins_accessions" from_work_dir="proteins_accession_numbers.txt" label="${tool.name} on ${on_string}: Proteins Accession Numbers">
+            <filter>(exporting_options['zip_conditional']['zip_output_boolean'] is False or (exporting_options['zip_conditional']['zip_output_boolean'] is True and exporting_options['zip_conditional']['export_reports_when_zip'] is True) ) and exporting_options['followup_conditional']['followup_conditional_selector'] == 'yes' and exporting_options['followup_conditional']['followup_export_accession_numbers'] != 'off' </filter>
+        </data>
+        <data format="fasta" name="output_proteins_sequences" from_work_dir="proteins_sequences.fasta" label="${tool.name} on ${on_string}: Proteins Sequences">
+            <filter>(exporting_options['zip_conditional']['zip_output_boolean'] is False or (exporting_options['zip_conditional']['zip_output_boolean'] is True and exporting_options['zip_conditional']['export_reports_when_zip'] is True) ) and exporting_options['followup_conditional']['followup_conditional_selector'] == 'yes' and exporting_options['followup_conditional']['followup_export_sequences'] != 'off' </filter>
+        </data>
+        <data format="txt" name="output_proteoforms" from_work_dir="proteins_proteoforms.txt" label="${tool.name} on ${on_string}: Proteins Proteoforms">
+            <filter>(exporting_options['zip_conditional']['zip_output_boolean'] is False or (exporting_options['zip_conditional']['zip_output_boolean'] is True and exporting_options['zip_conditional']['export_reports_when_zip'] is True) ) and exporting_options['followup_conditional']['followup_conditional_selector'] == 'yes' and exporting_options['followup_conditional']['followup_export_proteoforms'] != 'off' </filter>
```

Comment:
I think 

```
(exporting_options['zip_conditional']['zip_output_boolean'] is False or (exporting_options['zip_conditional']['zip_output_boolean'] is True and exporting_options['zip_conditional']['export_reports_when_zip'] is True) )
```

can be simplified to 

```
(exporting_options['zip_conditional']['zip_output_boolean'] is False or exporting_options['zip_conditional']['export_reports_when_zip'] is True)
```

and maybe a bit nicer

```
(not exporting_options['zip_conditional']['zip_output_boolean'] or exporting_options['zip_conditional']['export_reports_when_zip'])
```

But its all equivalent and maybe you prefer the longer version for clarity?

Reply:
Happy new year to you all :-)

@bernt-matthias , of course you are right about 
`
(exporting_options['zip_conditional']['zip_output_boolean'] is False or (exporting_options['zip_conditional']['zip_output_boolean'] is True and exporting_options['zip_conditional']['export_reports_when_zip'] is True) )
`
may be simplified, although regarding to filters I prefer to keep them as descriptive as possible due to it is really easy to make a mistake and not notice it for a long time.
However, I think the second option you suggest is clear enough and it will reduce the length of some filters (currently excessive). This will also be good for clarity purposes, so I would go for it.